src/main/kotlin/com/karumi/katagenda/servicelocator/AgendaServiceLocator.kt

```diff
@@ -0,0 +1,42 @@
+package com.karumi.katagenda.servicelocator
+
+import com.karumi.katagenda.common.repository.InMemoryDataSource
+import com.karumi.katagenda.common.repository.Repository
+import com.karumi.katagenda.domain.Agenda
+import com.karumi.katagenda.domain.Contact
+import com.karumi.katagenda.domain.repository.ContactsRepository
+import com.karumi.katagenda.ui.ContactsListPresenter
+import com.karumi.katagenda.ui.SysOutContactsListView
+import com.karumi.katagenda.usecase.AddContact
+import com.karumi.katagenda.usecase.GetContacts
+
+object AgendaServiceLocator {
```

Comment:
I would like to propose in another PR a ServiceLocator based on property delegation, to be more Kotliny, what do you think ? @flipper83 @Serchinastico

Reply:
thank you